extra/grafana/README.md

```diff
@@ -65,71 +65,75 @@ Dashboards may have limited functionality with earlier versions and some modific
-    # Specify the array endpoint from /metrics/array
-    metrics_path: /metrics/array
-    # Provide FlashArray authorization API token
-    authorization:
-      credentials: a12345bc6-d78e-901f-23a4-56b07b89012
-    # Provide parameters to pass the exporter the device to connect to. Provide FQDN or IP address
-    params:
-      endpoint: ['arrayname01.fqdn.com']
-
-    static_configs:
-    # Tell Prometheus which exporter to make the request
-    - targets:
-      - 10.0.2.10:9490
-      # Finally provide labels to the device.
-      labels:
-        # Instance should be the device name and is used to correlate metrics between different endpoints in Prometheus and Grafana. Ensure this is the same for each endpoint for the same device.
-        instance: arrayname01
-        # location, site and env are specific to your environment. Feel free to add more labels but maintain these three to minimize changes to Grafana which is expecting to use location, site and env as filter variables. 
-        location: uk
-        site: London
-        env: production
-
-# Repeat for the above for end points:
-# /metrics/volumes
-# /metrics/hosts
-# /metrics/pods
-# /metrics/directories
-
-# Repeat again for more Pure Storage FlashArrays
-```
+    This is an example [prometheus.yml](../prometheus/prometheus.yml) file.
+
+    Let's take a walkthrough an example of scraping the `/metrics/array` endpoint.
+
+    ```yaml
+    # Scrape job for one Pure Storage FlashArray scraping /metrics/array
+    # Each Prometheus scrape requires a job name. In this example we have structures the name `exporter_endpoint_arrayname`
+      - job_name: 'purefa_array_arrayname01'
+        # Specify the array endpoint from /metrics/array
+        metrics_path: /metrics/array
+        # Provide FlashArray authorization API token
+        authorization:
+          credentials: 11111111-1111-1111-1111-111111111111
+        # Provide parameters to pass the exporter the device to connect to. Provide FQDN or IP address
+        params:
+          endpoint: ['arrayname01.fqdn.com']
+
+        static_configs:
+        # Tell Prometheus which exporter to make the request
+        - targets:
+          - 10.0.2.10:9490
+          # Finally provide labels to the device.
+          labels:
+            # Instance should be the device name and is used to correlate metrics between different endpoints in Prometheus and Grafana. Ensure this is the same for each endpoint for the same device.
+            instance: arrayname01
+            # location, site and env are specific to your environment. Feel free to add more labels but maintain these three to minimize changes to Grafana which is expecting to use location, site and env as filter variables. 
+            location: uk
+            site: London
+            env: production
+
+      # Repeat for the above for end points:
+      # /metrics/volumes
+      # /metrics/hosts
+      # /metrics/pods
+      # /metrics/directories
```

Comment:
maybe add something here about 15 minutes scrape time for directories

Reply:
Resolved in commit [b3d29ee](https://github.com/PureStorage-OpenConnect/pure-fa-openmetrics-exporter/pull/87/commits/b3d29ee93cb226ac653ecbdbef90e65ed1c36196)